src/processing/style.ts

```diff
@@ -9,10 +9,22 @@ import INTERNAL_ATTRS from '../processing/dom/internal-attributes';
 import { isSpecialPage } from '../utils/url';
 import { URL_ATTR_TAGS } from './dom/attributes';
 import DomProcessor from './dom';
+import { Dictionary } from '../typings/common';
+
+function getTagsString (tagsDict: Dictionary<string>[]) {
+    const tags: string[] = [];
+
+    for (const key of Object.keys(tagsDict)) {
+        for (const tag of tagsDict[key])
+            tags.push(tag);
+    }
+
+    return tags.join().replace(/,/g, '|');
+}
 
 const SOURCE_MAP_RE                       = /\/\*\s*[#@]\s*sourceMappingURL\s*=[\s\S]*?\*\/|\/\/[\t ]*[#@][\t ]*sourceMappingURL[\t ]*=.*/ig;
 const CSS_URL_PROPERTY_VALUE_RE           = /(url\s*\(\s*(['"]?))([^\s]*?)(\2\s*\))|(@import\s+(['"]))([^\s]*?)(\6)/g;
-const TAGS_STRING                         = Object.values(URL_ATTR_TAGS).join().replace(/,/g, '|');
+const TAGS_STRING                         = getTagsString(URL_ATTR_TAGS as unknown as Dictionary<string>[]);
```

Comment:
Just use[ `values` from lodash](https://lodash.com/docs/4.17.15#values)

Reply:
It's a common code (shared between client and server). We cannot use here any libraries.
Also, need to rewrite the code to prevent crash in case if the `Object.keys` and `Array.join` methods are overridden on the client side.
```js

const objectKey = Object.prototype.keys;
const arrayJoin = Array.prototype.join;

function getTagsString (tagsDict: Dictionary<string>[]) {
    const tags: string[] = [];

    for (const key of objectKey.call(tagsDict)) {
        for (const tag of tagsDict[key])
            tags.push(tag);
    }

    return arrayJoin.call(tags.join).replace(/,/g, '|');
}
```